test/contracts/promises/promises-common/src/lib.rs

```diff
@@ -0,0 +1,85 @@
+#![no_std]
+
+pub use elrond_wasm::{Address, Vec};
+
+pub const PARENT_ADDRESS: [u8; 32] = [
```

Comment:
use this syntax instead, easier to read `b"\00\00\00\00..parent..............."`

Reply:
These contracts need revisiting indeed.